pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -126,6 +121,7 @@ type Cluster struct {
 	ownerRef            metav1.OwnerReference
 	csiConfigMutex      *sync.Mutex
 	isUpgrade           bool
+	arbiterMon          string
```

Comment:
I keep wondering if some of the recent addition might not be good candidates for ClusterInfo instead of expanding `Cluster`, thoughts?

Reply:
Good question. In this case, the arbiterMon only needs to be remembered in the `mon` package, so no need to pass it everywhere with the ClusterInfo

Reply:
Should I conclude this is something you are looking into?

Reply:
Actually I don't see a need to change. The stretch settings are passed around with the clusterSpec, and I don't think anyone outside the mons needs to know which mon is in which zone.